exercises/practice/strain/strain.spec.wren

```diff
@@ -0,0 +1,112 @@
+import "wren-testie/testie" for Testie, Expect
+import "./strain" for Strain
+
+Testie.test("Strain") { |do, skip|
+  do.describe("keep") {
+    do.test("keep on empty list returns empty list") {
+      var predicate = Fn.new { |x| true }
+      Expect.value(Strain.keep([], predicate)).toEqual([])
```

Comment:
more idiomatic to pass the predicate as a block (https://wren.io/functions.html#block-arguments):
```suggestion
      var actual = Strain.keep([]) { |x| true }
      Expect.value(actual).toEqual([])
```